docker-runner/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.8
+
+WORKDIR /go/src/github.com/zmap
+
+# Grab the currently-active version of the source
+ADD . zgrab2
+
+# This would instead grab it from the source repo
+# RUN go-wrapper download github.com/zmap/zgrab2
+
+WORKDIR /go/src/github.com/zmap/zgrab2
+
+RUN go get ./...
+RUN go get -t ./...
+
+# This should already be executable, but just in case...
+RUN chmod a+x ./docker-runner/entrypoint.sh
+
+# Build on the container
+RUN cd cmd/zgrab2 && go build && cd ../..
```

Comment:
I think you can do `go build cmd/zgrab2`, but not sure

Reply:
**Question** ([_pull request commenting with a purpose_](https://gist.github.com/andrewsardone/3fb9746e415d59784e97))

> I think you can do `go build cmd/zgrab2`, but not sure

Continuing off that, @justinbastress, could we delegate to `make clean zgrab2` or do we need to do a direct `go build`? Preference would be to keep things centralized in the `Makefile`.

Reply:
@dadrian indeed you can -- on Windows at least, `go build .\cmd\zgrab2` works.

@andrewsardone that's a good idea, My only concern is that it does add another dependency on the build, but `make` is probably a pretty universal pre-req; I'll confirm that the golang containers have it, and then I can start moving all of that build logic from .travis.yml into the Makefile.